model/src/w3gridmd.F90

```diff
@@ -1158,6 +1161,7 @@ SUBROUTINE W3GRID()
     NDSI   = 10
     NDSS   = 99
     NDSM   = 20
+    NDSM   = 21
```

Comment:
@mickaelaccensi, I'm not sure if this is related to the issue were seeing (maybe not, since as @JessicaMeixner-NOAA pointed out, its not in the mod_defs), nonetheless, likely one of these should be removed?